src/components/navBar/navBar.module.scss

```diff
@@ -0,0 +1,30 @@
+.navBar{
+  width: 100vw;
+  font-size: 16px;
```

Comment:
Please use `rem` for `font-size` so that when root HTML font size changes with screen width this changes too

Reply:
@whyDontI Interested to work on this.